### DISCUSSION TITLE
account blocked

### DISCUSSION THREAD

**Discussion by Saad-04** [vercel/community#5062] [Account Issue]
### Question

Vercel has restricted access to my GitHub account, preventing me from logging in. Upon attempting to sign in, I received a notification stating that my user account has been banned. I kindly request assistance in unbanning my main account (Saad-04) as this restriction is causing significant challenges for me during the deployment process. I appreciate your prompt attention to this matter. Thank you

**Comment by amyegan** [comment]
Hi @Saad-04. We can't safely discuss your account information in public. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
